src/main.zig

```diff
@@ -45,8 +49,10 @@ pub fn main() anyerror!void {
 }
 
 fn printPrompt(ally: *std.mem.Allocator) !void {
+    var buf = [_]u8{0} ** 100;
     const stdout = std.io.getStdOut();
-    try stdout.writer().writeAll("(shig)> ");
+    const cwd = std.os.getcwd(&buf);
```

Comment:
Peplace this with `std.process.getCwdAlloc(` that takes an allocator please. This way we won't have a problem if the path is more than 100 chars long.
Remember to free it.

Reply:
I should probably do the same in cd()

Reply:
It is better to have correct behavior than clean code.